src/test/java/org/example/controller/JobRoleControllerTest.java

```diff
@@ -50,4 +71,22 @@ void getAllJobRoles_shouldReturnError_whenServiceThrowsSQLException() throws SQL
 
         assertEquals(Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), response.getStatus());
     }
+
+    @Test
+    void getJobRoleById_shouldReturnJobRole() throws SQLException {
+        List<JobRoleDetailedResponse> expected = new ArrayList<>();
```

Comment:
This test will have to change to accommodate the fact that the dao/service methods should no longer return a list